### PR TITLE
Add missed PHPDoc blocks to AbstractAdmin and related interfaces

### PR DESCRIPTION
**Pull request by koninka**
Classes and interfaces:
- AbstractAdmin
- AdminInterface
- UrlGeneratorInterface

## Subject

I am targeting this branch, because I want to get more well-documented code. It's good practice to document code.

### CODE REVIEW

File: src/Admin/AbstractAdmin.php [hunk 1]
```diff
@@ -656,6 +662,9 @@ public function getExportFields()
         return $fields;
     }
 
+    /**
+     * {@inheritdoc}
+     */
```
Comment (VincentLanglet) [inline src/Admin/AbstractAdmin.php:667]: We recently removed all the inheritdoc we found.

This does not add any useful information, this is the default behavior of phpdoc

File: src/Admin/AdminInterface.php [hunk 2]
```diff
@@ -485,6 +488,9 @@ public function createObjectSecurity($object);
      */
     public function getParent();
 
+    /**
+     * @param AdminInterface $admin
+     */
```
Comment (VincentLanglet) [inline src/Admin/AdminInterface.php:493]: The typehint `self $admin` is enough to me.

File: src/Admin/UrlGeneratorInterface.php [hunk 3]
```diff
@@ -38,6 +38,9 @@ public function getRoutes();
      */
     public function getRouterIdParameter();
 
+    /**
+     * Set route generator.
+     */
```
Comment (VincentLanglet) [inline src/Admin/UrlGeneratorInterface.php:43]: This adds nothing. This is basically the name of the method.

**Review by VincentLanglet — COMMENTED** [review]
Thanks for your contribution.

Adding php-doc is a good practice indeed.
But it's only useful if it adds extra informations. IMHO

File: src/Admin/AbstractAdmin.php [hunk 4]
```diff
@@ -455,6 +455,9 @@ abstract class AbstractAdmin implements AdminInterface, DomainObjectInterface, A
      */
     protected $securityInformation = [];
 
+    /**
+     * @var array
```
Comment (core23) [inline src/Admin/AbstractAdmin.php:459]: Can you add more type information like key and value types

File: src/Admin/AbstractAdmin.php [hunk 5]
```diff
@@ -3263,22 +3269,47 @@ protected function configureQuery(ProxyQueryInterface $query): ProxyQueryInterfa
         return $defaultFilterValues;
     }
 
+    /**
+     * Configure fields for form view.
+     *
+     * @param FormMapper $form Object for simulating Form API
```
Comment (core23) [inline src/Admin/AbstractAdmin.php:3275]: This comment does not add any value

**Review by core23 — CHANGES_REQUESTED** [review]

**Comment by core23** [comment]
Thanks @koninka 

I merged your changes into this PR https://github.com/sonata-project/SonataAdminBundle/pull/6361

**Pull request closed without merging**